concent_api/core/tests/test_core_views.py

```diff
@@ -480,7 +480,7 @@ def test_send_task_to_compute_without_public_key_should_return_http_400(self):
 
             self._test_400_response(
                 response,
-                error_code=ErrorCode.MESSAGE_VALUE_WRONG_TYPE
+                error_message='Error in Golem Message.'
```

Comment:
Please make it a constant, i.e. `ERROR_IN_GOLEM_MESSAGE`

Reply:
:+1: done